src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -122,7 +122,7 @@ const NewWorkspaceModal = withDisplayName(
     const [groups, setGroups] = useState<string[]>([]);
     const [enhancedBucketLogging, setEnhancedBucketLogging] = useState(!!requireEnhancedBucketLogging);
     const [nameModified, setNameModified] = useState(false);
-    const [loading, setLoading] = useState(false);
+    const [loading, setLoading] = useState(true);
```

Comment:
When the component is mounted, `loadData` is called which will update `setLoading` to false once the Ajax call completes. Therefore make the default state `false` to avoid the dialog flashing when it first mounts.

Reply:
Nice. @msilva-broad this is something that should be added to `useAutoLoadedData`.